modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/multipart/MultipartUtil.java

```diff
@@ -48,7 +46,9 @@ public static String getFileName(Part part) {
 	public static boolean isMultipartBody(Parameter parameter) {
 		Class<?> clazz = parameter.getType();
 
-		return StringUtil.contains(clazz.getTypeName(), "MultipartBody");
+		String typeName = clazz.getTypeName();
```

Comment:
What is this fixing? Is StringUtil broken?

Reply:
It has into account a separator, by default a comma (,), to check if the substring it is contained between separators. Initially, the code was without using the StringUtil, and I changed, but I didn't realize about the current functionality of StringUtils.contains, so I just left as I was before, and I found it easier to understand that with a separator that we don't need

Reply:
Makes sense 👍